x/concentrated-liquidity/lp_test.go

```diff
@@ -359,52 +359,8 @@ func (s *KeeperTestSuite) TestWithdrawPosition() {
 			// for withdrawing a position.
 			sutConfigOverwrite: &lpTest{
 				lowerTick:     -1, // valid tick at which no position exists
-				expectedError: types.PositionNotFoundError{PoolId: 1, LowerTick: -1, UpperTick: 315000, JoinTime: defaultJoinTime, FreezeDuration: DefaultFreezeDuration},
-			},
-		},
-		"error: no position created (position exists but wrong freeze duration value)": {
-			// setup parameters for creation a pool and position.
-			setupConfig: baseCase,
-
-			// system under test parameters
-			// for withdrawing a position.
-			sutConfigOverwrite: &lpTest{
-				freezeDuration: DefaultFreezeDuration,
-				expectedError:  types.PositionNotFoundError{PoolId: 1, LowerTick: 305450, UpperTick: 315000, JoinTime: defaultJoinTime, FreezeDuration: DefaultFreezeDuration},
-			},
-			createPositionFreezeOverwrite: true,
-		},
-		"error: pool id for pool that does not exist": {
-			// setup parameters for creating a pool and position.
-			setupConfig: baseCase,
-
-			// system under test parameters
-			// for withdrawing a position.
-			sutConfigOverwrite: &lpTest{
-				poolId:        2, // does not exist
-				expectedError: types.PoolNotFoundError{PoolId: 2},
-			},
-		},
```

Comment:
This test still makese sense to me

Reply:
poold should not be used in this test case at all. I did however realize that `validatePositionUpdate` was still calling for it as an input parameter when all it needs is the positionId and expected remaining liquidity. I modified `validatePositionUpdate` to only take the params it needs here https://github.com/osmosis-labs/osmosis/pull/4695/commits/7dc15c2491548c7c435a9f1f90777e28343a8887